Makefile

```diff
@@ -17,3 +17,18 @@ schema-docs:
 		-cat aipscan \
 		-s aipscan \
 		-debug
+
+build:
+	# Install Node modules if not present
+	if [ ! -d "node_modules" ]; then \
+		npm install; \
+	fi
```

Comment:
Wouldn't you want to always run "npm install" or "npm ci"?

Reply:
Good point... npm doesn't re-install what's already installed.